.github/workflows/discord-bot.yml

```diff
@@ -0,0 +1,23 @@
+name: Notify Discord on PR Merge
```

Comment:
Do we want to notify on every PR merge. Sometimes the PR's are to fix typo's and small errors which might add noise